modules/core/src/main/scala/TextFormatting.scala

```diff
@@ -16,19 +16,53 @@
 
 package cue4s
 
-class TextFormatting(enabled: Boolean):
-  private def colored(msg: String)(f: String => fansi.Str) =
-    if enabled then f(msg).toString else msg
+object TextFormatting:
+  private def colored(msg: String)(f: String => fansi.Str) = f(msg).toString
```

Comment:
Given it no longer depends on enabled flag, this should be inlined at callsites and removes – it doesn't seem to serve any purpose anymore